docs/dev/install_scala_java_spark_maven.md

```diff
@@ -0,0 +1,66 @@
+# Install Scala && Java && Spark && Maven
+TiSpark is connector for Spark that enables you to use TiDB as the data source of Apache Spark, similar to other data sources.  Therefore, we need to install Scala and Spark. Our client for TiKV is written in Java, so we also need to install Java.
+## Install Scala 2.12
+We are now using Scala 2.12 . You can find out how to install Scala at  Scala 2.12.16.
```

Comment:
"You can find out how to install Scala at Scala 2.12.16." Don't get this sentence.